stylesheets/_global-wrapper.scss

```diff
@@ -185,23 +185,27 @@ ul.footer-nav {
   }
 }
 
-ul.social-nav {
+ul#social-nav {
   list-style: none;
-  width:65%;
-  float: left;
   margin: 0;
-  text-align:right;
-  padding-right:5%;
+  margin-left: 5%;
+  padding: .3rem 0;
```

Comment:
We're using PX on this site unless a module entirely uses `rems` - it's ideal not to mix the two back and forth.

If you must use rems, use the `remCalc(20)` mixin to convert 20px to rems